src/BlockTypes/CollectionStockFilter.php

```diff
@@ -77,16 +66,18 @@ protected function render( $attributes, $content, $block ) {
 	 */
 	private function get_stock_filter_html( $stock_counts, $attributes ) {
 		$display_style  = $attributes['displayStyle'] ?? 'list';
+		$show_counts    = $attributes['showCounts'] ?? false;
 		$stock_statuses = wc_get_product_stock_status_options();
 
 		// check the url params to select initial item on page load.
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce verification is not required here.
 		$selected_stock_status = isset( $_GET[ self::STOCK_STATUS_QUERY_VAR ] ) ? sanitize_text_field( wp_unslash( $_GET[ self::STOCK_STATUS_QUERY_VAR ] ) ) : '';
 
 		$list_items = array_map(
-			function( $item ) use ( $stock_statuses ) {
+			function( $item ) use ( $stock_statuses, $show_counts ) {
+				$label = $show_counts ? $stock_statuses[ $item['status'] ] . '(' . $item['count'] . ')' : $stock_statuses[ $item['status'] ];
```

Comment:
Look like we lose a space that's actually useful :D. 
<img width="612" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5423135/3f1e8a10-c383-4431-8e55-41d7f29d99db">

Reply:
Ha whoops. I will add a space before the first paren